pkg/network/ebpf/c/netns.h

```diff
@@ -14,7 +14,7 @@
 #endif
 
 #ifdef COMPILE_PREBUILT
-
+#include <net/sock.h>
```

Comment:
I think we were somehow implicitly relying on the header include order because this file (`netns.h`) is definitely missing the definition for the `struct sock` type for the prebuilt version, but I can't make sense of what was responsible for triggering this error.

Reply:
I'm wondering if that reveal a regression in npm
cc @hmahmood

Reply:
This is likely the result of some kernel header change that was including this file but now is not. I tried building this branch on ubuntu 20 and it builds find without this line. But on ubuntu 22 it fails.

Reply:
I don't think this would cause a regression though.